doctr/models/classification/mobilenet/pytorch.py

```diff
@@ -45,8 +49,16 @@
         'input_shape': (3, 32, 32),
         'rect_stride': ['features.2.block.1.0', 'features.4.block.1.0', 'features.9.block.1.0'],
         'vocab': VOCABS['french'],
+        'num_classes': len(VOCABS['french']),
         'url': None,
     },
+    'mobilenet_v3_small_orientation': {
+        'mean': (0.694, 0.695, 0.693),
+        'std': (0.299, 0.296, 0.301),
+        'input_shape': (3, 128, 128),
+        'num_classes': 4,
```

Comment:
with my previous suggestion, here the classes would be 0, 90, 180, 270

Reply:
yes ok